habitat-lab/habitat/datasets/rearrange/rearrange_generator.py

```diff
@@ -837,21 +850,23 @@ def initialize_sim(self, scene_name: str, dataset_path: str) -> None:
         """
         # Setup a camera coincident with the agent body node.
         # For debugging visualizations place the default agent where you want the camera with local -Z oriented toward the point of focus.
-        camera_resolution = [540, 720]
+        camera_cfg = self.cfg.agent.camera
         sensors = {
             "semantic": {
                 "sensor_type": habitat_sim.SensorType.SEMANTIC,
-                "resolution": [256, 256],
+                "resolution": camera_cfg.resolution,
                 "position": [0, 0, 0],
-                "orientation": [0, 0, 0.0],
+                "orientation": [camera_cfg.tilt_degrees * np.pi / 180, 0, 0.0],
+                "hfov": camera_cfg.hfov,
             },
         }
         if self._render_debug_obs:
             sensors["rgb"] = {
                 "sensor_type": habitat_sim.SensorType.COLOR,
-                "resolution": camera_resolution,
```

Comment:
Can we get this from camera_cfg?

Reply:
This is the debug camera config resolution, I thought it would be better to let it have a higher value and be independent of the agent's camera config.